### ISSUE TITLE
TextBuffer.on is deprecated.

### ISSUE THREAD

**Issue by fuczak** [Glavin001/atom-beautify#310]
Use TextBuffer::onWillSave instead. A TextBuffer instance is no longer provided as a callback argument.
```
TextBuffer.on (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1179:16)
Object.subscribeWith (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/emissary/lib/subscriber.js:28:32)
Object.subscribe (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/emissary/lib/subscriber.js:63:21)
<unknown> (/Users/damianprus/.atom/packages/atom-beautify/lib/beautify.coffee:297:18)
```

**Comment by Glavin001** [comment]
Duplicate of #289

**Comment by Glavin001** [comment]
Published to v0.26.5
